zap/src/test/java/org/zaproxy/zap/scan/filters/GenericFilterUtilityTest.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.scan.filters;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.parosproxy.paros.Constant;
+
+/** @author KSASAN preetkaran20@gmail.com */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Remove this and the `init` method, instead extend `WithConfigsTest` (should fix the build failure).

Reply:
yes, thanks alot for mentioning that. i was struggling with this. I was thinking of using JMockit to mock Static method.